### DISCUSSION TITLE
Product Reviews

### DISCUSSION THREAD

**Discussion by dwene** [medusajs/medusa#1368] [Feature Requests]
It would be really nice to have a plugin / direct feature support for product reviews. Has anyone done that yet?

**Comment by srindom** [comment]
Would you want the reviews to be stored in Medusa or do you know of any 3rd party product review services that it would make sense to integrate with?

**Reply by kawpii** [reply]
My preference is to store on Medusa and plugin support to store on 3rd party services. Storing on Medusa give users the data ownership, where third parties are more convenient.

**Comment by SGFGOV** [comment]
You can use yotpo, as a review service.. but ideally something as good as
yotpo which is open source will be ideal

Best Regards
Govind

On Sun, 26 Jun 2022, 08:22 kaushalyap, ***@***.***> wrote:

> My preference is to store on Medusa and plugin support to store on 3rd
> party services. Storing on Medusa give users the data ownership, where
> third parties are more convenient.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/medusajs/medusa/discussions/1368#discussioncomment-3024581>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXEQJHEV7CSRYIL7CZBPE6TVQ7AYBANCNFSM5TW4W4NQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 

This message (including any attachments) may contain confidential, 
proprietary, privileged and/or private
information. The information is 
intended to be for the use of the individual or entity designated above. If
you are not the intended recipient of this message, please notify the 
sender immediately, and delete the
message and any attachments. Any 
disclosure, reproduction, distribution or other use of this message or
any 
attachments by an individual or entity other than the intended recipient is 
prohibited.

**Comment by nicoestrada** [comment]
I came here to find this topic. Reviews is essential as a plugin imo. Yotpo is a good 3rd party solution if go that route

**Comment by edihasaj** [comment]
I did a plugin last year regarding this suggestion.

here is the link: https://github.com/edihasaj/medusa-plugin-reviews

**Reply by kulterryan** [reply]
Does this plugin still work with Medusa?

**Reply by SalahAdDin** [reply]
It seems to be not maintained anymore.

**Comment by dwene** [comment]
We also built a plugin for reviews. Haven't had the chance to build the admin for it, but it works great for us!

https://github.com/lambda-curry/medusa-plugin-product-reviews
